### PR TITLE
ghostwriter: switch to latest forked version & add autoupdate

### PR DESCRIPTION
**Pull request by hoilc**
https://github.com/wereturtle/ghostwriter/issues/367

> Windows Download
A volunteer has kindly stepped up to provide a Windows installer here:
https://github.com/michelolvera/vs-ghostwriter

### CODE REVIEW

**Comment by niheaven** [comment]
What's wrong with `persist`?

**Comment by hoilc** [comment]
> What's wrong with `persist`?

It was removed by mistake. 😂

File: bucket/ghostwriter.json [hunk 1]
```diff
@@ -1,17 +1,20 @@
 {
-    "homepage": "https://wereturtle.github.io/ghostwriter/",
+    "homepage": "https://github.com/michelolvera/vs-ghostwriter",
```
Comment (Ash258) [inline bucket/ghostwriter.json]: I really do not see a reson to change homepage to forked repository when it is only for binaires.
```suggestion
    "homepage": "https://wereturtle.github.io/ghostwriter/",
```

File: bucket/ghostwriter.json [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
-    "persist": "data",
     "shortcuts": [
         [
             "ghostwriter.exe",
             "Ghostwriter"
         ]
-    ]
+    ],
+    "persist": "data",
+    "checkver": "github",
```
Comment (Ash258) [inline bucket/ghostwriter.json]:
```suggestion
    "checkver": {
        "github": "https://github.com/michelolvera/vs-ghostwriter/"
    },
```

**Pull request merged**